akvo/akvo-flow
# Issue/2278, issue/2280, issue/2281 and  issue/2295 - overhaul the data cleaning report (Connect #2278, #2280, #2281, #2295)

#### Before the PR (what is the issue or what needed to be done)
Data  cleaning reports were hard to use.
#### The solution
Reorganise the repeatable question groups, putting their responses on separate sheets.
Also added headers for each group
#### Screenshots (if appropriate)

Test plan: 
 * make a data cleaning report.
 * change a value.
 * import it back into Flow and check that only that value was changed.
 * make a raw data report with advanced setting "for use with external system" (setting the useQuestionId option, which outputs the "variable names" as column headings).
* check that it still has the old format.


## Checklist
* [x] Connect the issue
* [ ] Test plan
